src/api/localResolvers/loanSearch.js

```diff
@@ -0,0 +1,61 @@
+import loanSearchStateQuery from '@/graphql/query/loanSearchState.graphql';
+// import gql from 'graphql-tag';
+
+// eslint-disable-next-line no-underscore-dangle
```

Comment:
I've noticed this eslint exclusion a lot, have we considered adding an exception to the config?

Reply:
Yea it does show up in a few places but always in the context of using `__typename`. So yea if we could make that an exception it would be good to clean out those eslint bypasses